include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf_dual_splitting.cpp

```diff
@@ -631,7 +644,13 @@ TimeIntBDFDualSplitting<dim, Number>::projection_step()
     if(this->print_solver_info() and not(this->is_test))
     {
       this->pcout << std::endl << "Explicit projection step:";
```

Comment:
Here, the same applies as above. The step is no longer "explicit" in case of HDIV.